demos/workbench/workbench.py

```diff
@@ -70,7 +70,25 @@ def add_text(history, text: str):
 def reset_app():
     app.chat_history = []
     app.N = 0
-    return [], None, "", gr.update(visible=False)
+
+    app.current_pdf_index = None
+    app.index_name = None
+    app.chunk_size = 500
+    app.chunking_technique = "Recursive Character"
+    app.chain = None
+    app.chat_history = None
+    app.N = 0
+    app.count = 0
+    app.use_semantic_cache = False
+    app.use_rerankers = False
+    app.top_k = 3
+    app.distance_threshold = 0.30
+    app.llm_temperature = 0.7
+    app.use_chat_history = False
+    app.use_semantic_router = False
+    app.use_ragas = False
```

Comment:
Isn't there a way to do this where we just re-init the ChatApp()? Not sure if there's any app state you are actually trying to retain. Could also wrap this into an app.reset() method for cleanliness?

Reply:
I'm working on the state management which will have that. This is a temp solution just for the reset functionality